templates/FFV_form_factors.cpp.in

```diff
@@ -104,17 +203,28 @@ namespace {
 * as specified in arXiv:0808.1819
 */
 
+//  TODO:  Several fixes still needed, 1/r is not helpful when r is infinite, 
+//  may need to just input mF and mS/mV and then calculate the ratios in-house.
+
+const double ratiolimit = 0.00001;
```

Comment:
Please rename this variable to `ratio_limit` or similar for readability. :)

Reply:
Done, thanks.